tpm2/policy_v0.go

```diff
@@ -374,7 +374,7 @@ func (p *keyDataPolicy_v0) UpdatePCRPolicy(alg tpm2.HashAlgorithmId, params *pcr
 		return xerrors.Errorf("cannot compute base PCR policy: %w", err)
 	}
 
-	pcrData.addRevocationCheck(trial, params.policyCounterName, params.policySequence+1)
+	pcrData.addRevocationCheck(trial, params.policyCounterName, params.policySequence)
```

Comment:
why this change?

Reply:
This PR moves incrementing the value earlier so this avoids a double increment.

Reply:
thanks, I followed the code a bit more and makes sense